### ISSUE TITLE
Update for Android 11 and Pixel 5

### ISSUE THREAD

**Issue by csgithub** [CaseyBakey/chaosp#13]
Any chance of this being update for Android 11 and Pixel 5

**Comment by CaseyBakey** [comment]
Hello and sorry for the delay.

I'm actually able to build CHAOSP on Android 11 for the following devices:
- Pixel 3 (blueline)
- Pixel 3a (sargo)
- Pixel 4a (sunfish)

The Pixel 5 isn't so different from the Pixel 4a and thus should work with my next update.

I switched to BiTGApps since opengapps didn't support Android 11 at this time (just realized today that they now support it).

Magisk v23.0 is working nicely, etc.

Nothing is commited yet, I'm cleaning my crappy/dirty scripts/modifications. Lot of stuff is hardcoded,  etc.

I had to rebase on latest RattlesnakeOS-stack build script template that changed quite a lot, etc.

I'm actually working on it on my free time and I hope to be able to ship it before the end of the month.
